### ISSUE TITLE
isinstance(m, nn.BatchNorm2d)

### ISSUE THREAD

**Issue by MonkeyD233** [valeoai/MVRSS#10]
Hello!
First of all, thank you for your open-source work. I found a potential issue while reading your code. Is there an indentation error at line 233 in `mvrss/learners/model.py`?

Since `isinstance(m, nn.Conv2d)` and `isinstance(m, nn.BatchNorm2d)` are mutually exclusive, the `elif isinstance(m, nn.BatchNorm2d)` condition after `m.bias is None` will always evaluate to False. As a result, the BatchNorm2d initialization code block is effectively dead code that is never executed.
